purchases/src/main/kotlin/com/revenuecat/purchases/WebPurchaseRedemption.kt

```diff
@@ -0,0 +1,9 @@
+package com.revenuecat.purchases
+
+/**
+ * Represents a web redemption link, that can be redeemed using [Purchases.redeemWebPurchase]
+ */
+@ExperimentalPreviewRevenueCatPurchasesAPI
+class WebPurchaseRedemption internal constructor(
+    internal val redemptionToken: String,
```

Comment:
should this just be `token`?

Reply:
Hmm could be... It's internal so we can really change it whenever... I think it might be better to leave it as is so we know the type of token it is, not a strong opinion though